Adyen/UI/Form/FormViewController.swift

```diff
@@ -203,18 +230,36 @@ open class FormViewController: UIViewController, AdyenObserver, PreferredContent
 
     // MARK: - Private
 
-    private func addFormView() {
-        view.addSubview(formView)
-        view.backgroundColor = style.backgroundColor
-        formView.backgroundColor = style.backgroundColor
-        formView.adyen.anchor(inside: view.safeAreaLayoutGuide)
+    private func addSubviews() {
+        if !scrollDisabled {
```

Comment:
Maybe better cognitive ease to start with `if isScrollingDisabled` instead of negative